### ISSUE TITLE
Modal: close icon hover contrast

### ISSUE THREAD

**Issue by danethurber** [pluralsight/design-system#1583]
Use a Button component. It should be a11'ble.

> The "x" close element has no color change when you hover on it. Expected color ratio is at least 3:1.
> Please follow next link to find the correct colors for hover state to reach the expected ratio: https://contrast-finder.org/
